sdks/java/io/kafka/build.gradle

```diff
@@ -49,6 +50,9 @@ dependencies {
   compile library.java.vendored_guava_26_0_jre
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile project(":sdks:java:expansion-service")
+  permitUnusedDeclared project(":sdks:java:expansion-service")
+  compile "org.apache.kafka:kafka-clients:2.4.1"
```

Comment:
kafka-clients should not be a compile dependency. And I don't see an error for it:

```
Execution failed for task ':sdks:java:io:kafka:analyzeClassesDependencies'.
> Dependency analysis found issues.
  usedUndeclaredArtifacts:
   - io.confluent:kafka-schema-registry-client:5.3.2@jar
   - org.apache.avro:avro:1.8.2@jar
  unusedDeclaredArtifacts:
   - beam.sdks.java:expansion-service:2.29.0-SNAPSHOT@jar
```